runtime.md

```diff
@@ -16,6 +16,7 @@ There is no requirement that it be unique across hosts.
 The ID is provided in the state because hooks will be executed with the state as the payload.
 This allows the hooks to perform cleanup and teardown logic after the runtime destroys its own state.
 * **`pid`**: (int) is the ID of the main process within the container, as seen by the host.
+The pid MAY be excluded or set to a value indicating empty or null if it is not seen by the host.
```

Comment:
Pointers are [our usual pattern for optional fields][1].  I don't think a zero PID is clearly enough “no-op in the spec” to qualify for the non-pointer-ok exception listed there.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/style.md#optional-settings-should-have-pointer-go-types